main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+)
+
+// Converts a JSON file containing people and ages to a human-readable txt file
+func main() {
+
+	type Person struct {
+		FirstName string
+		Age       int
+	}
+
+	var people []Person
+
+	data, err := ioutil.ReadFile("./testdata/people.json")
+	if err != nil {
+		fmt.Println("error:", err)
+	}
+
+	err = json.Unmarshal(data, &people)
+	if err != nil {
+		fmt.Println("error:", err)
+	}
+
+	for i := range people {
```

Comment:
`range` is my favourite thing in Go! It's so versatile. With a slice, we can get successive _index_ values, as here, or if we're interested only in the _elements_, we can get just those:

```suggestion
	for _, p := range people {
```

Now `p` will be, successively, each `Person` in the slice.